depthai_nodes/ml/parsers/lane_detection.py

```diff
@@ -44,7 +44,7 @@ def __init__(
         row_anchors: List[int] = None,
         griding_num: int = None,
         cls_num_per_lane: int = None,
-        input_shape: Tuple[int, int] = (288, 800),
+        input_size: Tuple[int, int] = None,
```

Comment:
We need to be careful as this requires also updates to the public NNArchives on HubAI as it is a metadata parameter name change

Reply:
The change is not breaking as the `input_shape` is currently not defined based on the `head_metadata`. I think we should actually remove those parameters from the archives!

Reply:
I see. In that case this was a bit of a bug because if we exported this model with different input shape the automatic ParserGenerator wouldn't work correctly as input shape would not be changed from default right?

Reply:
Yes, I think that's right. The `input_shape` was always (288, 800) by default unless changed with the `setInputShape` setter.